mods/ctf_pvp_engine/ctf_flag/flag_func.lua

```diff
@@ -210,6 +210,14 @@ ctf_flag = {
 					minetest.chat_send_player(name, "You are at peace with this team!")
 					return
 				end
+				
+				local glitcher = players_glitching[name]
```

Comment:
What's actually stored in `players_glitching[name]` is a pos. You might want to rename `glitcher` to `pos` or something to eliminate confusion.